pages/configuration/elasticsearch.rst

```diff
@@ -478,6 +478,7 @@ After you've removed the index template, new indices will only have the original
     }
   }
 
+Additional information on Elasticsearch Index Templates can be found in the official `Elasticseach Documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html>`: 
```

Comment:
would you please write the link following:

```
`Elasticseach Documentation <https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html>`_
```

In addition we refer linking to a specific version and not current as Elastic tend to change how the product work between versions and we mostly refer to the latest 'official' supported version in Graylog. What is 6.6 at the time of writing.

Reply:
Updated the link ( and some more to 6.6) hope that is ok.